src/routes/github/subscription-deferred-install/github-subscription-deferred-install-get.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response } from "express";
+import { extractSubscriptionDeferredInstallPayload } from "services/subscription-deferred-install-service";
+import { hasAdminAccess } from "services/subscription-installation-service";
+
+const escapeHtml = (unsafe: string) => {
+	return unsafe
+		.replace(/&/g, "&amp;")
+		.replace(/</g, "&lt;")
+		.replace(/>/g, "&gt;")
+		.replace(/"/g, "&quot;")
+		.replace(/'/g, "&#039;");
+};
+
+export const GithubSubscriptionDeferredInstallGet = async (req: Request, res: Response) => {
+	const payload = await extractSubscriptionDeferredInstallPayload(req);
+	const { githubToken, installation } = res.locals;
+
+	if (!await hasAdminAccess(githubToken, installation.jiraHost, payload.gitHubInstallationId, req.log, payload.gitHubServerAppIdPk)) {
+		req.log.warn("Not an admin");
+		res.status(401).json({
+			error: "Must be a GitHub admin"
+		});
+		return;
+	}
+
+	const payloadObjForReview = {
+		... payload,
+		jiraHost: res.locals.jiraHost,
+		gitHubUrl: res.locals.gitHubAppConfig.hostname,
+		... (res.locals.gitHubAppConfig.gitHubAppId ? {
+			gheClientId: res.locals.gitHubAppConfig.clientId,
+			gheAppId: res.locals.gitHubAppConfig.gheAppId
+		} : { })
+	};
+
+	res.send(
+		`
```

Comment:
Okay, lol, just see this, you just hate handlerbar don't you?

Reply:
Can we convert this to a template? We're already using Handlebars and it's better to stick with a consistent pattern of using it throughout our codebase.

Reply:
Also, can you provide a screenshot of what this looks like?

Reply:
That doesn't suppose to have any UI, ok?.....